src/core/mcis/nlb.go

```diff
@@ -126,17 +126,22 @@ type NLBHealthCheckerInfo struct {
 	KeyValueList []common.KeyValue `json:"keyValueList"`
 }
 
-type SpiderHealthInfo struct {
+type SpiderNLBHealthInfoWrapper struct {
+	Healthinfo SpiderNLBHealthInfo
+}
+
+type SpiderNLBHealthInfo struct {
 	AllVMs       *[]common.IID
 	HealthyVMs   *[]common.IID
 	UnHealthyVMs *[]common.IID
 }
 
 type TBNLBTargetGroup struct {
-	Protocol string   `json:"protocol" example:"TCP"` // TCP|HTTP|HTTPS
-	Port     string   `json:"port" example:"22"`      // Listener Port or 1-65535
-	MCIS     string   `json:"mcis" example:"mc"`
-	VMs      []string `json:"vms"`
+	Protocol string `json:"protocol" example:"TCP"` // TCP|HTTP|HTTPS
+	Port     string `json:"port" example:"22"`      // Listener Port or 1-65535
+
+	VmGroupId string   `json:"vmGroupId"`
```

Comment:
```suggestion
	VmGroupId string  `json:"vmGroupId" example:"group"`
```
api 예시를 추가해두었습니다. :)